### ISSUE TITLE
Fail to load BouncyCastle.Crypto, Version = 1.8.1.0

### ISSUE THREAD

**Issue by diegorvieira17** [vivet/GoogleApi#228]
I had problems with DistanceMatrix api in my .Net Framework 4.5 project. When I exec the call for 

`var response = GoogleMaps.DistanceMatrix.QueryAsync(request).Result;`

I had the error below:

`Could not load file or assembly 'BouncyCastle.Crypto, Version = 1.8.1.0, Culture = neutral, PublicKeyToken = 0e99375e54769942' or one of its dependencies. The system cannot find the file specified.`

I searched in project dependencies and the BrounceCastle.Cripto is there.

Can someone helpe me to solve this problem?

![image](https://user-images.githubusercontent.com/7876106/101183542-6a9cf980-362e-11eb-91bf-e3b570954790.png)

**Comment by vivet** [comment]
Hi
It could be a problem with the NuGet. It's a long time since I worked with 4.5. 

I will check it out

**Comment by vivet** [comment]
Hi
I have tried creating a new console project and do a Distance Matrix request using digital singed key, and I couldn't reproduce you issue. I will need your sample code, if I am to try and reproduce what is happening for you.

**Comment by vivet** [comment]
Hi
I hope you solved the issue. 
Since I haven't heard back from you, I will close the issue

**Comment by kbalint** [comment]
Hello, I got the following running on 4.8. I don't know where BouncyCastle 1.8.10 originates from, Nuget says 1.8.9 is the latest available.
```

Warning		Found conflicts between different versions of "BouncyCastle.Crypto" that could not be resolved.
There was a conflict between "BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942" and "BouncyCastle.Crypto, Version=1.8.10.0, Culture=neutral, PublicKeyToken=0e99375e54769942".
    "BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942" was chosen because it was primary and "BouncyCastle.Crypto, Version=1.8.10.0, Culture=neutral, PublicKeyToken=0e99375e54769942" was not.
    References which depend on "BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942" [D:\Projects\[BigProject]\packages\BouncyCastle.1.8.9\lib\BouncyCastle.Crypto.dll].
        D:\Projects\[BigProject]\packages\BouncyCastle.1.8.9\lib\BouncyCastle.Crypto.dll
          Project file item includes which caused reference "D:\Projects\[BigProject]\packages\BouncyCastle.1.8.9\lib\BouncyCastle.Crypto.dll".
            BouncyCastle.Crypto, Version=1.8.9.0, Culture=neutral, PublicKeyToken=0e99375e54769942
            D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\[BigProject.Component].dll
        D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\GoogleApi.dll
          Project file item includes which caused reference "D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\GoogleApi.dll".
            D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\[BigProject.Component].dll
    References which depend on "BouncyCastle.Crypto, Version=1.8.10.0, Culture=neutral, PublicKeyToken=0e99375e54769942" [D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\BouncyCastle.Crypto.dll].
        D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\[BigProject.Component].dll
          Project file item includes which caused reference "D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\[BigProject.Component].dll".
            D:\Projects\[BigProject]\[BigProject.Component]\bin\Debug\[BigProject.Component].dll			

```

**Comment by vivet** [comment]
Hi
I have re-opened the issue
GoogleApi uses 1.8.1.3. 
I am currently working on a version 4 of the library, and I will try to do some tests for net framework 4.5 and 4.8.

**Comment by kbalint** [comment]
Hi, thanks! I think the problem is that you use
`    <PackageReference Include="Portable.BouncyCastle" Version="1.8.1.3" />
`
while my project has the not-so-portable (?) BouncyCastle, not the portable one, which is at 1.8.9 for me,.

**Comment by vivet** [comment]
I have updated the Portable.BouncyCastle to the highest version, that GoogleApi can support. 
I have recently worked with net framework 4.8, and in complex solutions this mess often happens. Try look for overrides in the project files and trace the use of the different packages.

I am not sure if there is more I can do, as the problem originates outside this Library. 
Hope you figure it out

will close issue.
